test/e2e/lua/dynamic_configuration.go

```diff
@@ -51,16 +51,6 @@ var _ = framework.IngressNginxDescribe("Dynamic Configuration", func() {
 		err := f.NewEchoDeploymentWithReplicas(1)
 		Expect(err).NotTo(HaveOccurred())
 
-		err = f.WaitForNginxConfiguration(func(cfg string) bool {
-			servers, err := dns.GetSystemNameServers()
-			Expect(err).NotTo(HaveOccurred())
-			ips := []string{}
-			for _, server := range servers {
-				ips = append(ips, fmt.Sprintf("\"%v\"", server))
-			}
-			return strings.Contains(cfg, "configuration.nameservers = { "+strings.Join(ips, ", ")+" }")
-		})
-
```

Comment:
As you can see I wasn't asserting on `err` here and the check was timing out after 5 minutes, and this was happening before every test. Therefore e2e test runtime doubled.